src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/subscriptionBox.jsx

```diff
@@ -10,41 +10,41 @@ import {Flex} from 'grid-emotion';
 import withOrganization from 'app/utils/withOrganization';
 import SentryTypes from 'app/sentryTypes';
 
-class SubscriptionBox extends React.Component {
+export class SubscriptionBox extends React.Component {
   static propTypes = {
     resource: PropTypes.string.isRequired,
-    disabled: PropTypes.bool.isRequired,
+    disabledFromPermissions: PropTypes.bool.isRequired,
+    webhookDisabled: PropTypes.bool.isRequired,
     checked: PropTypes.bool.isRequired,
     onChange: PropTypes.func.isRequired,
     organization: SentryTypes.Organization,
   };
 
-  constructor(...args) {
-    super(...args);
-    this.state = {
-      checked: this.props.checked,
```

Comment:
Don't need this state field since we have a prop for `checked` which is set with `onChange`.